CHANGELOG.md

```diff
@@ -22,14 +23,29 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ### Changed
 - Upgraded Hive version to 3.1.2 (was 2.3.7).
 
+## [5.3.1] - TBD
```

Comment:
You also need to have all of these mentioned again under `6.1.0` as that's where they will appear in this branch.